### PR TITLE
CLI snapshots updated in README.md.

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#39 
The video has been uploaded to [asciinema](https://asciinema.org).
Although I had updated preface frame of the video to a frame that contains more information about `pyrgg`, It seems it hasn't change in [defualt picture](https://asciinema.org/a/352310.png).
It may need some more time to updated.

The screenshot has been uploaded to [uupload.ir](http://uupload.ir/).
I can send it to you so that you can upload it to your webpage (It has 1 year time limit for each uploaded file).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- CLI snapshots updated
```
Comment (sepandhaghighi) [inline CHANGELOG.md:8]: add `### Changed`

File: README.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ Note that it may run on lower end equipment though good performance is not guara
 
 <div align="center">
 
-<a href="https://asciinema.org/a/141608" target="_blank"><img src="https://asciinema.org/a/141608.png" /></a>
+<a href="https://asciinema.org/a/352310" target="_blank"><img src="https://asciinema.org/a/352310.png" /></a>
```
Comment (sepandhaghighi) [inline README.md:124]: Nice work, but I suggest to choose another preview for this image.
You can select this preview in asciinema.org website

File: README.md [hunk 3]
```diff
@@ -201,7 +201,7 @@ or send an email to [info@pyrgg.ir](mailto:info@pyrgg.ir "info@pyrgg.ir").
 	- [Frog](https://github.com/AndrewStallman/Frog "Frog")
 	- [CuSha](https://github.com/farkhor/CuSha "CuSha")
 <div align="center">
-<img src="http://www.shaghighi.ir/pyrgg/images/random.png">
+<img src="http://uupload.ir/files/rag1_screenshot_from_2020-08-08_11-23-40.png">
```
Comment (sepandhaghighi) [inline README.md:204]: uuload.ir is not relibale.
Please use this link : 
https://www.pyrgg.ir/images/random.png

File: README.md (hunk 2 above)
Comment (sadrasabouri) [reply README.md:124]: Thank you.
But I'm not sure what do you mean <b>this</b>?
It seems your comment doesn't contain any link.

Comment (sepandhaghighi) [reply README.md:124]: I talk about preface frame of the video

**Pull request merged**